packages/ui-extensions/src/api.ts

```diff
@@ -0,0 +1,8 @@
+export interface ExtensionInstance {
+  readonly id: string;
```

Comment:
What is this `id` for?

Reply:
This is mostly just to have a field on this object. In checkout extensions, we use this "instance" object to reflect to the extension details about itself, including metadata (the extesnion version, script URL, etc), and runtime information (whether the extension is rendered, what capabilities it has). I'd like to have a place for that information to live in this package, so I added this type. I didn't want to get into any of those other concepts that will likely live here, so I just opted for a single field that is kind of a baseline expectation: a unique identifier for the running "instance".

If you feel it's too confusing, I can remove it for now. Checkout does have this concept, so I will eventually add it back, but I don't need it for getting the editor extension point off the ground.

Reply:
We have this concept in Admin as well it's tied to the version `uuid` we get from GraphQL. Can a single extension version have multiple `id`?

Reply:
Yes, but probably not in the admin? In checkout, you can both install a single extension in multiple spots, and a single extension may render multiple times, if it targets a piece of UI that repeats (currently, this is just the `CartLineDetails::RenderAfter` extension point.

Reply:
I see - I think we can just leave it as `id` then. Admin can just use the `uuid` for this value